lua/telescope/builtin/git.lua

```diff
@@ -72,7 +72,8 @@ git.commits = function(opts)
 end
 
 git.stash = function(opts)
-  opts.entry_maker = vim.F.if_nil(opts.entry_maker, make_entry.gen_from_git_stash())
+  opts.show_branch = vim.F.if_nil(opts.show_branch, true)
```

Comment:
we need to document this option here: https://github.com/nvim-telescope/telescope.nvim/blob/27294d73e4562cbc83c9c488b4618f012deade7e/lua/telescope/builtin/init.lua#L204-L207
with `@field`